src/views/Ecosystem/Ecosystem.js

```diff
@@ -160,14 +160,14 @@ export default function Ecosystem() {
                 <div className="App-card-row">
                   <div className="label">Link</div>
                   <div>
-                    <a href="https://t.me/GMXPositions" target="_blank" rel="noopener noreferrer">
-                      https://t.me/GMXPositions
+                    <a href="https://t.me/MYCPositions" target="_blank" rel="noopener noreferrer">
+                      https://t.me/MYCPositions
```

Comment:
Will there be a Telegram account/group created with this URL? If not could be removed

Reply:
ill remove all these ecosystem links